.travis.yml

```diff
@@ -45,9 +46,11 @@ before_script:
   -DINSTALL_QPOISSON_RECON_PLUGIN=ON
   -DINSTALL_QSRA_PLUGIN=ON
   -DINSTALL_QSSAO_PLUGIN=ON
+  -DBUILD_TESTING=ON
   ..
 script:
 - if [[ $TRAVIS_BRANCH != "beta_builds" ]]; then make -j2; fi
```

Comment:
To fix the clang test issue (https://github.com/travis-ci/travis-ci/issues/8613), add the following line before running ctest:

`- if [[ $CXX == "clang++" ]]; then export LD_LIBRARY_PATH=/usr/local/clang/lib:$LD_LIBRARY_PATH; fi`